cli/main.go

```diff
@@ -45,8 +46,9 @@ func buildVersionString() string {
 	}
 	fmt.Fprint(versionsWriter, "\n")
 	fmt.Fprintf(versionsWriter, "reference values for %s platform:\n", platforms.AKSCloudHypervisorSNP.String())
-	fmt.Fprintf(versionsWriter, "\truntime handler:\tcontrast-cc-%s\n", manifest.TrustedMeasurement[:32])
-	fmt.Fprintf(versionsWriter, "\tlaunch digest:\t%s\n", manifest.TrustedMeasurement)
+	if refValues, err := json.MarshalIndent(manifest.EmbeddedReferenceValuesJSON, "", " "); err == nil {
```

Comment:
Sorry I did not add enough context. What I meant to say is to do something like this:

```go
json.MarshalIndent(manifest.embeddedReferenceValues, "\t\t\t\t", "  ")
``` 

And this should marshal the object, not the byte array.

Reply:
Oh. This makes sense now..